### PR TITLE
Add several spelling corrections

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

**Comment by vikivivi** [comment]
@luzpaz Discard or moving `bu->by` to rare might have a successful checks

**Comment by luzpaz** [comment]
Done

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -1815,6 +1817,9 @@ analyticaly->analytically
 analyticly->analytically
 analyzator->analyzer
 analyzies->analysis, analyses, analyzes,
+analzye->analyze
+analzyed->analyzed
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:1821]:
```suggestion
analzyed->analyzed
analzyer->analyzer
```

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -1815,6 +1817,9 @@ analyticaly->analytically
...
+analzying->analyzing
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:1822]: Can you add the GB (s) variants of these too, as separate typo-correction pairs.

File: codespell_lib/data/dictionary.txt [hunk 3]
```diff
@@ -4909,6 +4922,7 @@ candadate->candidate
 candadates->candidates
 candiate->candidate
 candiates->candidates
+candidats->candidates
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:4925]: Also this one (although it's a valid French word... but that might explain the occurrences of it)
```suggestion
candidat->candidate
candiate->candidate
candiates->candidates
candidats->candidates
```

File: codespell_lib/data/dictionary.txt [hunk 4]
```diff
@@ -7413,6 +7429,7 @@ constaints->constraints
 constallation->constellation
 constallations->constellations
 constanly->constantly
+constans->constants, constant,
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:7432]:
```suggestion
constans->constants, constant, Constance,
```

File: codespell_lib/data/dictionary.txt [hunk 5]
```diff
@@ -17459,6 +17493,7 @@ inititalisation->initialisation
 inititalisations->initialisations
 inititalization->initialization
 inititalizations->initializations
+inititalize->initialize
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:17496]: Can you also separately add the ise versions.

File: codespell_lib/data/dictionary.txt [hunk 6]
```diff
@@ -20984,6 +21021,7 @@ newslines->newlines
 newthon->newton
 newtork->network
 Newyorker->New Yorker
+nexting->nesting
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:21024]: Maybe:
```suggestion
nexting->nesting, texting,
```

File: codespell_lib/data/dictionary.txt [hunk 7]
```diff
@@ -21233,6 +21271,7 @@ numebrs->numbers
 numer->number
 numeraotr->numerator
 numerbering->numbering
+numercial->numerical
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:21274]:
```suggestion
numercial->numerical
numercially->numerically
```

File: codespell_lib/data/dictionary.txt [hunk 8]
```diff
@@ -29405,6 +29453,7 @@ stiky->sticky
 stil->still
 stilus->stylus
 stingent->stringent
+stip->strip
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:29456]:
```suggestion
stip->strip, stop,
```

**Review by peternewman — CHANGES_REQUESTED** [review]
Some suggestions

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: 69e4305

File: codespell_lib/data/dictionary.txt (hunk 5 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt:17506]: 01b1a64

**Comment by peternewman** [comment]
Can you resolve the conflicts too please...

File: codespell_lib/data/dictionary_rare.txt [hunk 9]
```diff
@@ -16,6 +16,7 @@ bellow->below
 bloc->block
 blocs->blocks
 bodgy->body
+bu->by
```
[the remark this answers is not in the document]
Comment (vikivivi) [reply codespell_lib/data/dictionary_rare.txt:19]: Yes, it was failing the tests when it was in `dictionary.txt`. See https://github.com/codespell-project/codespell/pull/2322/commits/c6c624bc414cdf2213ec7598a6797e4b24b2881e

**Comment by luzpaz** [comment]
@vikivivi please re-review, thanks!

File: codespell_lib/data/dictionary.txt [hunk 10]
```diff
@@ -1825,6 +1827,10 @@ analoguously->analogously
 analogus->analogous
 analouge->analogue
 analouges->analogues
+analsye->analyse
+analsyed->analysed
+analsyer->analyser
+analsying->analysing
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:1833]: @luzpaz if you do not mind adding these
```text
analsyers->analysers
analsyes->analyses
analsyt->analyst
analsyts->analysts
```
* https://github.com/search?q=analsyers&type=code
* https://github.com/search?q=analsyes&type=code
* https://github.com/search?q=analsyt&type=code
* https://github.com/search?q=analsyts&type=code

File: codespell_lib/data/dictionary.txt [hunk 11]
```diff
@@ -1841,6 +1847,10 @@ analyticaly->analytically
 analyticly->analytically
 analyzator->analyzer
 analyzies->analysis, analyses, analyzes,
+analzye->analyze
+analzyed->analyzed
+analzyer->analyzer
+analzying->analyzing
 ananlog->analog
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:1854]:
```text
analzyers->analyzers
analzyes->analyzes
```
* https://github.com/search?q=analzyers&type=code
* https://github.com/search?q=analzyes&type=code

File: codespell_lib/data/dictionary.txt [hunk 12]
```diff
@@ -9958,6 +9980,9 @@ describibg->describing
 describng->describing
 describtion->description
 describtions->descriptions
+descrice->describe
+descriced->described
+descrices->describes
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:9985]:
```text
descricing->describing
```
* https://github.com/search?q=descricing&type=code

File: codespell_lib/data/dictionary.txt [hunk 13]
```diff
@@ -14875,6 +14904,7 @@ followin->following
 followind->following
 followint->following
 followng->following
+followwing->following
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:14907]:
```text
followwings->followings
```
* https://github.com/search?q=followwings&type=code

File: codespell_lib/data/dictionary.txt [hunk 14]
```diff
@@ -15310,9 +15341,11 @@ funcions->functions
 funciton->function
 funcitonal->functional
 funcitonality->functionality
+funcitonally->functionally
 funcitoned->functioned
 funcitoning->functioning
 funcitons->functions
+functiion->function
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:15348]:
```text
functiional->functional
functiionality->functionality
functiionally->functionally
functiioning->functioning
functiions->functions
```
* https://github.com/search?q=functiional&type=code
* https://github.com/search?q=functiionality&type=code
* https://github.com/search?q=functiionally&type=code
* https://github.com/search?q=functiioning&type=code
* https://github.com/search?q=functiions&type=code

File: codespell_lib/data/dictionary.txt [hunk 15]
```diff
@@ -26685,6 +26731,7 @@ repet->repeat
 repetative->repetitive
 repete->repeat
 repeteadly->repeatedly
+repetetive->repetitive
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:26734]:
```text
repetetion->repetition
repetetions->repetitions
```
* https://github.com/search?q=repetetion&type=code
* https://github.com/search?q=repetetions&type=code

File: codespell_lib/data/dictionary.txt [hunk 16]
```diff
@@ -27029,6 +27076,7 @@ resignement->resignment
 resistable->resistible
 resistence->resistance
 resistent->resistant
+resitance->resistance
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:27079]:
```text
resitances->resistances
```
* https://github.com/search?q=resitances&type=code

File: codespell_lib/data/dictionary.txt [hunk 17]
```diff
@@ -31339,6 +31392,7 @@ tendacy->tendency
 tendancies->tendencies
 tendancy->tendency
 tennisplayer->tennis player
+tentaive->tentative
```
Comment (vikivivi) [inline codespell_lib/data/dictionary.txt:31395]:
```text
tentaively->tentatively
```
* https://github.com/search?q=tentaively&type=code

**Review by vikivivi — CHANGES_REQUESTED** [review]

**Comment by luzpaz** [comment]
@vikivivi added changed in 1caf9f4

**Comment by vikivivi** [comment]
@luzpaz  I suggest you withdraw the below which you have added in [1caf9f4](https://github.com/codespell-project/codespell/pull/2322/commits/1caf9f4ebb2ad8f29fae45f8f4e0900b06a24866)
These are camelCase unit for (height, horizontal, Hessian, horsepower, hard stop, etc)
```text
unith->unit
uniths->units
```

**Pull request merged**